### DISCUSSION TITLE
How to deserialize from JSON?

### DISCUSSION THREAD

**Discussion by beppe9000** [RehanSaeed/Schema.NET#62] [Q&A]
Here it says it can deserialize but can't find how...
https://www.nuget.org/packages/Schema.NET.NullHandling/3.6.1

**Comment by RehanSaeed** [comment]
Wrong package. Someone has created a clone:

https://www.nuget.org/packages/Schema.NET/

You can use `JsonConvert`. See unit tests in the project.
